pkg/config/options.go

```diff
@@ -39,6 +57,8 @@ func (o *Options) Bind(fs *flag.FlagSet) {
 	fs.StringVar(&o.LogLevel, "log-level", "info", "Level at which to log output.")
 	fs.StringVar(&o.Org, "org", "", "Limit repos affected to those in this org.")
 	fs.StringVar(&o.Repo, "repo", "", "Limit repos affected to this repo.")
+
+	fs.BoolVar(&o.determineChanges, "determine-changes", false, "If true, compare changes with the main branch")
```

Comment:
nit: maybe `--only-process-changes` or something? This does a lot more than just determine them